client/src/api/block_builder/input_selection/new/remainder.rs

```diff
@@ -57,16 +59,45 @@ impl InputSelection {
         None
     }
 
+    pub(crate) fn remainder_amount(&self) -> Result<u64> {
```

Comment:
Maybe rename, because this only returns a potential required remainder amount for an output with native tokens, but we could require/have a remainder amount without native tokens?

Reply:
Is `required_remainder_amount` good for you ?

Reply:
No, I think it's still missing the difference for the remainder amount without native tokens

Reply:
Maybe `minimum_remainder_amount` makes sense instead.
Otherwise do you have a suggestion ?

Reply:
Ah yeah ofc, sorry I'm in meetings so I didn't think this through, will do it later + add a test.

Reply:
There is already `not_enough_storage_deposit_for_remainder()` so it seems to be handled fine already somewhere, just the naming of the functions sounds like it would also handle this case, but it doesn't

Reply:
Yeah there is a test for that but I don't think there is a test when another available input should be selected to increase an already existing remainder amount that is less than the rent, and I don't think this case is currently handled at all

Reply:
Can confirm the issue exist, will fix and then I think the name makes sense because it covers both